olive/data/container/dummy_data_container.py

```diff
@@ -18,17 +18,21 @@ class DummyDataContainer(DataContainer):
         dummy_data_config = DataConfig(
             name="dummy",
             type="DummyDataContainer",
-            "load_dataset_config"={
-                "params": {
+            load_dataset_config=DataComponentConfig(
+                params={
                     "input_names": metric.user_config.input_names,
                     "input_shapes": metric.user_config.input_shapes,
                     "input_types": metric.user_config.input_types,
                 }
-            }
+            ),
+            dataloader_config=DataComponentConfig(
+                # TODO(shaahji): Remove this override once DataConfig transition is complete
+                params={ "batch_size": None }
+            )
         )
     """
 
     default_components_type: ClassVar[dict] = {
         DataComponentType.LOAD_DATASET.value: "dummy_dataset",
-        DataComponentType.DATALOADER.value: "skip_dataloader",
+        DataComponentType.DATALOADER.value: "default_dataloader",
```

Comment:
`no_auto_batch_dataloader` takes the same effect with `default_dataloader` and `params={ "batch_size": None }`
![image](https://github.com/microsoft/Olive/assets/13343117/504245a8-1c3b-4c3d-ae79-0082ae192317)

Reply:
Done!